zebra-chain/src/block/height.rs

```diff
@@ -48,6 +48,14 @@ impl Height {
     /// `Height::MAX.0` can't be used in match range patterns, use this
     /// alias instead.
     pub const MAX_AS_U32: u32 = Self::MAX.0;
+
+    /// The maximum expiration Height that is allowed in coinbase transactions
+    /// previous to Nu5 and in non-coinbase transactions from Nu5 activation height
+    /// and above.
+    ///
+    /// TODO: This is currently the same as `Height::MAX` but that change in #1113.
+    /// Remove this TODO when that happens.
+    pub const MAX_COINBASE_EXPIRATION_PRE_NU5: Height = Height(499_999_999);
```

Comment:
Confusing Name: The name says that the limit only applies to pre-NU5 coinbase transactions, but it also applies to non-coinbase transactions.

We usually name constants after the general rule, and then document any exceptions in the comments.

```suggestion
    pub const MAX_EXPIRY_HEIGHT: Height = Height(499_999_999);
```